test/unit/grammar/cases.verb-tenses.js

```diff
@@ -0,0 +1,144 @@
+// See https://www.englishpage.com/verbpage/
+// Not all verb forms are represented here, but I think I got the ones likely to make trouble.
+
+export const verbTenses = {
+    "Verb tenses": {
+        Simple: {
+            Present: [
+                {
+                    descr: "Regular verb",
+                    in: "She speaks English",
+                    out: "They speak English"
+                },
+                {
+                    descr: "Regular with adverb",
+                    in: "He only speaks English",
+                    out: "They only speak English"
+                },
+                {
+                    descr: "Do statement",
+                    in: "She does not speak English",
+                    out: "They do not speak English"
+                },
+                {
+                    descr: "Do question",
+                    in: "Does he speak English?",
+                    out: "Do they speak English?"
+                },
+                {
+                    descr: "Do question, negated",
+                    in: "Doesn't she speak English?",
+                    out: "Do they not speak English?"
```

Comment:
I would write "Don't they" to keep the contraction, but this is completely readable. I would mention this somewhere else if I knew where to put it. I see other tense tests with similar expansion of contractions.

Reply:
I totally agree - see #92. I'll add a comment to this effect.